### PR TITLE
Cve 2019 14857

### PR DESCRIPTION
**Pull request by darunesh-RH**

### CODE REVIEW

File: 2019/14xxx/CVE-2019-14857.json [hunk 1]
```diff
@@ -48,14 +48,35 @@
                 "url": "https://bugzilla.redhat.com/show_bug.cgi?id=CVE-2019-14857",
```
Comment (cve-team) [inline 2019/14xxx/CVE-2019-14857.json]: Invalid source.

File: 2019/14xxx/CVE-2019-14857.json [hunk 2]
```diff
@@ -48,14 +48,35 @@
...
-            }
+            },
+	    {
+		    "url": "https://github.com/zmartzone/mod_auth_openidc/commit/5c15dfb08106c2451c2c44ce7ace6813c216ba75",
+		    "name": "https://github.com/zmartzone/mod_auth_openidc/commit/5c15dfb08106c2451c2c44ce7ace6813c216ba75",
+		    "refsource": "CONFIRM"
+	},
+	{
+		"url": "https://github.com/zmartzone/mod_auth_openidc/commit/ce37080c6aea30aabae8b4a9b4eea7808445cc8e",
+		"nmae": "https://github.com/zmartzone/mod_auth_openidc/commit/ce37080c6aea30aabae8b4a9b4eea7808445cc8e",
```
Comment (iamleot) [inline 2019/14xxx/CVE-2019-14857.json]: There is a typo here: `nmae` should be `name`.

File: 2019/14xxx/CVE-2019-14857.json [hunk 3]
```diff
@@ -47,15 +47,36 @@
             {
```
Comment (cve-team) [inline 2019/14xxx/CVE-2019-14857.json]: Invalid source.

File: 2019/14xxx/CVE-2019-14857.json [hunk 4]
```diff
@@ -56,7 +56,7 @@
 	},
 	{
 		"url": "https://github.com/zmartzone/mod_auth_openidc/commit/ce37080c6aea30aabae8b4a9b4eea7808445cc8e",
-		"nmae": "https://github.com/zmartzone/mod_auth_openidc/commit/ce37080c6aea30aabae8b4a9b4eea7808445cc8e",
+		"name": "https://github.com/zmartzone/mod_auth_openidc/commit/ce37080c6aea30aabae8b4a9b4eea7808445cc8e",
 		"refsource": "CINFIRM"
```
Comment (iamleot) [inline 2019/14xxx/CVE-2019-14857.json]: ...and it seems there is a typo here too (probably should be `CONFIRM` not `CINFIRM`).

**Pull request merged**